tests/containers/single_container_webui.pm

```diff
@@ -1,24 +1,23 @@
-use Mojo::Base qw(openQAcoretest);
+use qw(openQAcoretest);
```

Comment:
What was the intent here?

Reply:
oops, that should still have the Mojo::Base, or at least "base". Fixed